src/App.tsx

```diff
@@ -26,70 +27,139 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType !== SortType.NONE) {
+    switch (sortType) {
```

Comment:
Is this check necessary?
You have a default case in switch, so nothing will be sorted

```suggestion
    switch (sortType) {
```